### PR TITLE
Redesign/sign in page

### PR DESCRIPTION
**Pull request by Tanyashinke**

### CODE REVIEW

File: frontend/src/app/libs/auth-pages/sign-in/sign-in.component.html [hunk 1]
```diff
@@ -1,32 +1,53 @@
...
+			</div>
+
+			<div class="sign-in-page__container-divider">
+				<div class="sign-in-page__divider"></div>
+				<p class="sign-in-page__or">{{ "SIGN_IN.OR" | translate }}</p>
+				<div class="sign-in-page__divider"></div>
+			</div>
+			<form [formGroup]="signInFormGroup" (ngSubmit)="submit()" class="sign-in-page__form">
+				<div>
+					<app-input [inputValidators]="emailValidators" [theme]="darkTheme"
```
Comment (maximwow70) [inline frontend/src/app/libs/auth-pages/sign-in/sign-in.component.html:29]: Please, provide each attribute on separated line.

File: frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts [hunk 2]
```diff
@@ -39,25 +55,30 @@ export class SignInComponent implements OnInit, OnDestroy {
...
 	}
 
-	public get emailInputErrorMessage(): string {
-		if ((this.signInFormGroupControl.email.touched || this.submitted) && !this.signInFormGroupControl.email.valid) {
-			const emailInputErrorMessage: string[] = Object.keys(this.signInFormGroupControl.email.errors);
-			return emailInputErrorMessage[0];
+	public emailValidator(control: AbstractControl): ValidationErrors | null {
+		const maxLength: number = 128;
+		const emailPattern: RegExp = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+		if (control && Boolean(control.value) && control.value.length >= maxLength ||
```
Comment (maximwow70) [inline frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts:67]: Looks like we don't need to check twice on `control && Boolean(control.value)`

File: frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts [hunk 3]
```diff
@@ -69,6 +90,20 @@ export class SignInComponent implements OnInit, OnDestroy {
 	}
 
 	public submit(): void {
+		this.recaptchaV3Service.execute('signUpUser')
```
Comment (maximwow70) [inline frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts:93]: until destroy missing?

Comment (Tanyashinke) [reply frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts]: I have added  .pipe(takeUntil(this.destroy$)) here.

File: frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts (hunk 2 above)
Comment (Tanyashinke) [reply frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts]: I deleted redundant check.

File: frontend/src/app/libs/auth-pages/sign-in/sign-in.component.html (hunk 1 above)
Comment (Tanyashinke) [reply frontend/src/app/libs/auth-pages/sign-in/sign-in.component.html]: I fixed formatting.

**Review by maximwow70 — APPROVED** [review]

**Pull request merged**